parm/forecast/regional/nems.configure.atm_ocn.tmp

```diff
@@ -39,7 +36,7 @@ OCN_attributes::
   hycom_arche_output = .false.
   hyc_esmf_exp_output = .true.
   hyc_esmf_imp_output = .true.
-  hyc_impexp_file = dummy_file
+  hyc_impexp_file = nems.configure
```

Comment:
@danrosen25, Just want to double check that, after your latest [fix](https://github.com/hafs-community/HYCOM-src/commit/09d6265f1fc59e4f60cd7a87ab577d4373b9d5d4), do we need to change hyc_impexp_file from nems.configure back to dummy_file? Also, how about the ocn_export/import_fields sections below? I assume they are harmless here, even when building the forecast model without the CMEPS=Y option, correct?
Thanks! -Bin

Reply:
You need to use 'dummy_file' if you're not providing a file with ocn_export_fields and ocn_import_fields. It's relevant to all runs with HYCOM turned on and coupled (both CMEPS and directly coupled HYCOM)

Reply:
@danrosen25, I went ahead and conducted four tests with the combinations of the CMEPS build option (on vs off) and this hyc_impexp_file option (dummy_file vs nems.configure). They all produced identical results for the direct coupling configuration (cpl_ocean=2).

Reply:
@danrosen25 Just saw your new comment in the thread. With that, I will change hyc_impexp_file into dummy_file for the cpl_ocean=2 (direct coupling) configuration.